Assignment1/Extensions.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  Extensions.swift
+//  Prob
+//
+//  Created by Dar Dar on 04.10.2023.
+//
+
+import Foundation
```

Comment:
I was wondering if i should move the extensions from the ViewController to this file, so ViewController is cleaner. Should I move them to another file, or it s fine to have the extensions in the ViewController?

Reply:
well, tableViewDataSource and delegate should be kept in the controller's file because they refer to the UITableView, which is in that view controller. The extension for photo downloading can be moved to this file because it's a general extension and you may need to use it somewhere else.